source/ci_source/providers/Travis.ts

```diff
@@ -83,6 +83,10 @@ export class Travis implements CISource {
     // TODO: This only supports travis.org
     return `https://travis-ci.org/${this.env.TRAVIS_REPO_SLUG}/jobs/${this.env.TRAVIS_JOB_ID}`
   }
+
+  get commitHash(): string {
+    return this.env.TRAVIS_COMMIT
```

Comment:
`TRAVIS_COMMIT: The commit that the current build is testing.`
From: https://docs.travis-ci.com/user/environment-variables/